api/HTMLMeterElement.json

```diff
@@ -26,7 +26,10 @@
             "version_added": "11"
           },
           "safari": {
-            "version_added": "5"
+            "version_added": "6"
```

Comment:
Where did the original "5" come from, and can you check when `<meter>` itself was supported?

Reply:
The original data comes from the wiki migration.  The element itself was introduced in Safari 6, which matches this new data.

Reply:
Thanks for checking!